### PR TITLE
Prevent bringing less Pokemon in Doubles and Triples battles

### PR DESCRIPTION
**Pull request by xInstruct**
Caused a crash earlier, but this should prevent it

### CODE REVIEW

**Comment by xInstruct** [comment]
actually, theres a better way to do this

File: sim/dex-formats.ts [hunk 1]
```diff
@@ -686,6 +686,14 @@ export class DexFormats {
 				if (subformat.hasValue === 'positive-integer') {
 					if (parseInt(value) <= 0) throw new Error(`In rule "${ruleSpec}", "${value}" must be positive.`);
 				}
+				if (subformat.id === 'pickedteamsize' || subformat.id === 'maxteamsize') {
+					if (format.gameType === 'doubles' && parseInt(value) < 2) {
+						throw new Error(`In a doubles battle, at least 2 Pokemon is needed.`);
+					}
+					if (format.gameType === 'triples' && parseInt(value) < 3) {
+						throw new Error(`In a triples battle, at least 3 Pokemon is needed.`);
```
Comment (AnnikaCodes) [inline sim/dex-formats.ts:694]:
```suggestion
						throw new Error(`In a doubles battle, at least 2 Pokemon are needed.`);
					}
					if (format.gameType === 'triples' && parseInt(value) < 3) {
						throw new Error(`In a Triples battle, at least 3 Pokemon are needed.`);
```

Comment (AnnikaCodes) [reply sim/dex-formats.ts:694]: This should probably be in the `else if (subformat.hasValue === 'integer' || subformat.hasValue === 'positive-integer')` block? That's where the rest of the number validation is

Comment (xInstruct) [reply sim/dex-formats.ts]: Did it a much better way, this is now irrelevant

**Comment by Zarel** [comment]
You're missing a check for `maxTeamSize >= minTeamSize` and `maxTeamSize >= pickedTeamSize`.

**Pull request closed without merging**